### DISCUSSION TITLE
Welcome to PartiQL Discussions!

### DISCUSSION THREAD

**Discussion by rchowell** [partiql/partiql-lang#1] [Announcements]
## 👋 Welcome!
  
The PartiQL Team is happy to announce that we're hosting Github Discussions on partiql-lang!

**Comment by johnedquinn** [comment]
## How to Subscribe
For those who have stumbled open this first post: if you are interested in subscribing to PartiQL Discussions, please click on `partiql-lang` (or click [here](https://github.com/partiql/partiql-lang)) > `Watch` > `Custom` > `Discussions` to receive notifications on new discussions! You can also subscribe to individual posts by clicking the `Subscribe` button on any post.
